projects/dot-rules/src/lib/rule-engine.ts

```diff
@@ -139,8 +140,9 @@ export class RuleEngineComponent {
         this._rsrcCache = {};
         this.status = null;
 
-        this.ruleViewService.message.subscribe((message: string) => {
-            this.globalError = message;
+        this.ruleViewService.message.subscribe((dotRuleMessage: DotRuleMessage) => {
```

Comment:
Do we need a takeUntil here?

Reply:
done https://github.com/dotCMS/core-web/pull/1291/commits/81cfcdd04df51d307c1dcb84dfae91f278bb23c0